modules/core/src/v2/wallet.ts

```diff
@@ -352,6 +353,7 @@ export interface SendManyOptions {
   recipients?: {
     address: string;
     amount: string | number;
+    feeLimit?: string;
```

Comment:
Why is this option specified per-recipient?

Reply:
this is because each contract call could have different fee limit